spec/mailers/user_mailer_spec.rb

```diff
@@ -501,6 +503,24 @@ def expect_email_body_to_have_help_and_contact_links
   end
 
   describe '#verify_by_mail_letter_requested' do
```

Comment:
A/N: After changing the locale handling logic, tests in this context are now failing, but only when run with the whole suite, which suggests something is getting memoized and shouldn't be.

Reply:
Fixed by @jmhooper's change here: [commit: clean up the locale helper implementation](https://github.com/18F/identity-idp/pull/11351/commits/1fd39b7c3dea88ced4d2ba05eee83023231f313d)


The important thing to note is that the Rspec runner only makes one instance of the UserMailer, so the locale wasn't being set correctly (it had a value, instead of `nil`, which is what we expect for email protocols; correct me if I misunderstood!) _and_ the changes to the locale weren't getting cleaned up after each test, thus why running a specific `context` or  example would run fine, but running the whole spec caused failures.